src/components/text-editor/examples/text-editor-basic.tsx

```diff
@@ -0,0 +1,35 @@
+import { LimelTextEditorCustomEvent } from '@limetech/lime-elements';
```

Comment:
is this really needed? 🤔

Reply:
There was some version of Stencil that started complaining if you used `CustomEvent` instead of the correct type output by Stencil (in this case `LimelTextEditorCustomEvent`). But, no it's not strictly necessary. I think you could generally use `CustomEvent` instead, and you could of course just type the event as `any` in the event handler. But this is the correct type, and this is an example where we try to influence the reader to use good coding style, so I think we should keep it 😊